net.displaycal.DisplayCAL.json

```diff
@@ -0,0 +1,199 @@
+{
+    "id": "net.displaycal.DisplayCAL",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk//20.08",
+    "runtime-version": "20.08",
+    "rename-desktop-file": "displaycal.desktop",
+    "rename-icon": "displaycal",
+    "copy-icon": true,
+    "command": "displaycal",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
```

Comment:
```suggestion
        "--socket=fallback-x11",
```

Reply:
Wasn't this already addressed in https://github.com/flathub/flathub/pull/1999#issuecomment-803381360 ?

Reply:
we need X11. DisplayCal run external program (Argyllcms as built into the flatpak) that use X11.
Wayland here is used for the UI in Gtk3.

Reply:
> Wasn't this already addressed in [#1999 (comment)](https://github.com/flathub/flathub/pull/1999#issuecomment-803381360) ?

This is open since December, I rarely look at previous reviews when submission is considered ready by authors. Also GitHub is hardly Gerrit…

@hfiguiere Noted.

Reply:
added comment.